### PR TITLE
Add healthcheck for Substrate <-> Ethereum

### PR DESCRIPTION
**Pull request by eddyjlsh**

### CODE REVIEW

File: utils/utils.ts [hunk 1]
```diff
@@ -51,20 +55,49 @@ export function getEnvironmentRequiredVars() {
...
     ? process.env.MNEMONIC_POLK
     : " oh oh";
 
+  const ethereumWsUrl = process.env.ETHEREUM_WEBSOCKET_URL
+    ? process.env.ETH_APP_ADDRESS
+    : "ws://localhost:8545";
+
+  const substrateWsUrl = process.env.SUBSTRATE_WEBSOCKET_URL
+    ? process.env.ETH_APP_ADDRESS
+    : "ws://localhost:9944";
```
Comment (goncer) [inline utils/utils.ts:83]: I think that this guy should match with API_URL, shouldn't it? It should be the target test environment. So maybe we can reuse it from above.

File: utils/utils.ts [hunk 2]
```diff
@@ -51,20 +55,49 @@ export function getEnvironmentRequiredVars() {
...
     ? process.env.MNEMONIC_META
     : " oh oh";
+
   const mnemonicPolkadot = process.env.MNEMONIC_POLK
     ? process.env.MNEMONIC_POLK
     : " oh oh";
 
+  const ethereumWsUrl = process.env.ETHEREUM_WEBSOCKET_URL
+    ? process.env.ETH_APP_ADDRESS
+    : "ws://localhost:8545";
```
Comment (goncer) [inline utils/utils.ts:79]: This guy is the bridge, right? 
in UI-test.yml, we modify depending on the environment the env. variables.  I just modified it yesterday, so you may need to update this branch with the latest main.
Do you mind adding those?

name: Set Configuration - Staging
name: Set Configuration - Develop

File: test/healthcheck/ethereum-substrate.test.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
+/* eslint-disable prettier/prettier */
```
Comment (goncer) [inline test/healthcheck/ethereum-substrate.test.js:1]: groups! 
ci ?

File: utils/utils.ts (hunk 2 above)
Comment (eddyjlsh) [reply utils/utils.ts:75]: I merged main but I'm not 100% sure what you mean - I can see the github actions variables to do with `name: Set Configuration` but what do I need to change in `utils`?

**Comment by eddyjlsh** [comment]
This PR will fail because the environment variables for ethereum contracts are not in place yet.

File: utils/utils.ts (hunk 2 above)
Comment (goncer) [reply utils/utils.ts:75]: ohh sorry, in utils you don't need to change anything, its in /.github/workflows/ui-test.yml where you need to place the env.Variables values.

**Review by goncer — CHANGES_REQUESTED** [review]
small reminder: 
Plz add the workflow changes to this PR :)

**Comment by goncer** [comment]
While reviewing, plz fix the conflict!

File: .github/workflows/ui-test.yml [hunk 4]
```diff
@@ -79,13 +79,19 @@ jobs:
...
         if: contains(github.event.client_payload.environment, 'develop')
         run:  |
           echo 'UI_URL=https://develop.mangata.finance/' >> $GITHUB_ENV
           echo 'API_URL=wss://develop.mangatafinance.cloud:9944' >> $GITHUB_ENV
-        
+          echo 'ETHEREUM_WEBSOCKET_URL=ws://localhost:8545' >> $GITHUB_ENV
+          echo 'ETH_APP_ADDRESS=0xDd514baA317bF095ddBa2C0a847765fEb389c6a0' >> $GITHUB_ENV
+          echo 'ETH_20_APP_ADDRESS=0x00e392C04743359e39f00CD268A5390D27Ef6b44' >> $GITHUB_ENV
+
       - name: Set Configuration - Integration
```
Comment (goncer) [inline .github/workflows/ui-test.yml:95]: Integration ?

File: .github/workflows/ui-test.yml [hunk 5]
```diff
@@ -79,13 +79,19 @@ jobs:
         run:  |
           echo 'UI_URL=https://staging.mangata.finance/' >> $GITHUB_ENV
           echo 'API_URL=wss://staging.mangatafinance.cloud:9944/' >> $GITHUB_ENV
-  
+          echo 'ETHEREUM_WEBSOCKET_URL=ws://localhost:8545' >> $GITHUB_ENV
```
Comment (goncer) [inline .github/workflows/ui-test.yml:82]: Hey @eddyjlsh ,
I can see 3 issues,
* Integration config is missing.
* Values are duplicated for develop and staging.
* Values does not match with the zip that contains all this info. Am I missing anything?

**Review by goncer — CHANGES_REQUESTED** [review]

**Pull request merged**